src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -396,33 +401,21 @@ function createModel(
                 if (apiVersionIndex !== -1) {
                     const apiVersionInOperation =
                         op.Parameters[apiVersionIndex];
-                    if (
-                        apiVersionInOperation.DefaultValue?.Value &&
-                        !apiVersions.has(
-                            apiVersionInOperation.DefaultValue?.Value
-                        )
-                    ) {
-                        apiVersions.add(
-                            apiVersionInOperation.DefaultValue.Value
-                        );
-                    } else {
-                        if (
-                            apiVersionInOperation.Location ===
+                    if ((!apiVersionInOperation.DefaultValue?.Value ||
```

Comment:
This logic is complex, could you add some comments?

Reply:
added